### ISSUE TITLE
'RegressionCatalog.RegressionTrainers' does not contain a definition for 'Ols'

### ISSUE THREAD

**Issue by ooples** [dotnet/docs#33924]
Running the code example [from this link](https://learn.microsoft.com/en-us/dotnet/api/microsoft.ml.mklcomponentscatalog.ols?view=ml-dotnet#microsoft-ml-mklcomponentscatalog-ols(microsoft-ml-regressioncatalog-regressiontrainers-system-string-system-string-system-string)) gives the following exception:

`'RegressionCatalog.RegressionTrainers' does not contain a definition for 'Ols' and no accessible extension method 'Ols' accepting a first argument of type 'RegressionCatalog.RegressionTrainers' could be found `

**Comment by gewarren** [comment]
@luisquintanilla Can you take a look at this issue about the ML.NET API docs?

**Comment by ooples** [comment]
@luisquintanilla @gewarren Is there any update on this error?

**Comment by luisquintanilla** [comment]
Yup. I'll take a look.

**Comment by luisquintanilla** [comment]
@ooples do you have the https://www.nuget.org/packages/Microsoft.ML.Mkl.Components/ package installed?

**Comment by ooples** [comment]
@luisquintanilla Yes that fixed the error so I would definitely edit the example code to reflect the needed package. Is there a full list of regression trainers since some of them seem to require different libraries such as this one or the LightGBM trainer. Also why isn't the OLS trainer in its own separate package like the rest?
